### PR TITLE
Workaround for #1359

### PR DESCRIPTION
**Pull request by fxha**
| Q                | A
| ---------------- | ---
| License          | MIT

### Description

Workaround for #1359 to prevent drop shadow flickering and sometimes application flickering by temporary disabling GPU.

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,26 @@
 ## [unrelease]
 
+**:warning:Breaking Changes:**
+
+- Temporary disabled hardware acceleration (GPU) on Linux with Wayland due to rendering issues with the frameless window.
+
 **:cactus:Feature**
 
+- Allow cycle through tabs via shortcuts
```
Comment (Jocs) [inline .github/CHANGELOG.md]: Please add the issue number to every changes.

**Comment by Jocs** [comment]
I think we can update changelog one time when we release a new version through `git commit log` or `project done issues`.

**Comment by fxha** [comment]
The best way would be a bot that can read the changelog message from the PR comment like Electron does. A changelog should be for the user and commit headings are a bad idea.

**Comment by Jocs** [comment]
>A changelog should be for the user and commit headings are a bad idea.

Do you mean use the headings as changelog description is a bad idea?

**Comment by fxha** [comment]
Yes, commit headings and messages are for developers and changelog for the end-user. E.g. `Workaround for #1359` vs `Temporary disabled hardware acceleration (GPU) on Linux with Wayland due to rendering issues with the frameless window.`

**Comment by Jocs** [comment]
First, I didn't think of a good solution. Secondly, I don't think the priority is too high. Every time I release a new version, it is OK to update the changelog from the project or commit log. After that, we still need to think of a better solution.

**Comment by Jocs** [comment]
Please solve the conflict before merge it.

**Pull request merged**